server.js

```diff
@@ -30,9 +32,7 @@ app.get('*', (req, res) => {
     fetchData(state).then((data) => {
       const props = {data, state, req};
       const html = React.renderToString(<Root {...props} />);
-      let template = fs.readFileSync(path).toString();
-      template = template.replace('__HTML__', html);
-      template = template.replace('__DATA__', JSON.stringify(data));
+      const content = template.replace('__HTML__', html).replace('__DATA__', JSON.stringify(data));
```

Comment:
do you want to assign the output of the  first `replace` to a temporary variable? or happy with just new lining it?

```
let content = template.replace('__HTML__', html);
content =  content.replace('__DATA__', JSON.stringify(data));
```
or
```
const content = template.replace('__HTML__', html)
  .replace('__DATA__', JSON.stringify(data));
```